pkgs/misc/drivers/logiops/0_3_3.nix

```diff
@@ -0,0 +1,58 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  pkg-config,
+  cmake,
+  glib,
+  udev,
+  libevdev,
+  libconfig,
+}:
+
+stdenv.mkDerivation (
+  oldAttrs: {
```

Comment:
No indent for the top level `mkDerivation` and function please. `nixpkgs-fmt` will honor this and not touch it.